# DesiredStateMsg

float64[3] x
float64[3] x_dot
float64[3] x_2dot
float64 yaw
float64 yaw_rate
